lib/util.js

```diff
@@ -27,9 +27,9 @@ function isEmpty(object) {
 
 function getValues(object) {
   let result = [];
-  for (const key in object) {
+  for (const [, value] of Object.entries(object)) {
```

Comment:
I believe `Object.values` might be better suited here?

Reply:
@tiaanduplessis ye I agree - browser support for `Object.entries` and `Object.values` appears to be the same, so we might as well use `.values`:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

An aside - do you know what the best practice is for using "new" Javascript specs in modules? 

I generally iterate over `Object.keys` to avoid adding polyfills, at the expense of having to manually fetch values out of the object.

Is it up to the library to provide support for older browsers, or is it up to the user to add polyfills where necessary? 

I imagine the responsible thing is to indicate which polyfills are necessary for older browsers when newer features are used in a module.

Reply:
@larrybotha I'm not quite sure what is the current best practice for Svelte related modules, but I'd probably go with the user needing to add polyfilling where required and then indicating newer features used in the module README for full transparency.

The reason being that using core-js via babel will allow the user the flexibility to configure exactly what they want via  browserlist and since Svelte does not compile down to older versions of JS out of the box the extra configuration will need to be added in any case to handle modern syntax and polyfiliing of popular browser APIs like Promises.

What do you think?

Reply:
@tiaanduplessis ye, sounds good to me!